### PR TITLE
DX-1617: Checkout standard

### PR DESCRIPTION
**Pull request by arebra**
* Sequence diagrams are not done
 * Seamless view events will probably be revisited when Vitaliia gives the green light. 
 * Screenshots of shipping options are not included yet.

First draft. Let loose.

### CODE REVIEW

File: checkout/v3/mac/index.md [hunk 1]
```diff
@@ -4,30 +4,18 @@ title: Introduction
 estimated_read: 3
 hide_from_sidebar: false
 description: |
-  **MAC** is the Checkout option for merchants with their own consumer
+  **MAC** is the Checkout for merchants with their own consumer
```
Comment (asbjornu) [inline checkout/v3/mac/index.md:7]: I think it's wise to spell out `MAC` whenever there's room for it, so we avoid the confusion with Apple Macintosh as much as possible.

```suggestion
  **Merchant Authenticated Consumer** is the Checkout for merchants with their own consumer
```

File: checkout/v3/set-up.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Here, Gabriellas sensational setup sentences will appear.
```
Comment (asbjornu) [inline checkout/v3/set-up.md:1]: 😄

**Review by asbjornu — CHANGES_REQUESTED** [review]
👏🏼 Great stuff! But please take some time to review whether many of the requests, responses, and field tables can be sourced from includes instead of possibly duplicated inline.

File: _includes/payment-order-checkout-standard.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
+{% capture documentation_section %}{%- include documentation-section.md -%}{% endcapture %}
+{% capture features_url %}{% include documentation-section-url.md href='/features' %}{% endcapture %}
+{% assign operation_status_bool = include.operation_status_bool | default: "false" %}
+
+{:.code-view-header}
+**Request**
```
[the remark this answers is not in the document]
Comment (arebra) [reply _includes/payment-order-checkout-standard.md:6]: I guess. I followed suit after Nina initially divided them when we started, and I just followed suit for this one. Could we leave them separated for this release and create a fancy if-fest for a December touch-up?

[the remark this answers is not in the document]
Comment (asbjornu) [reply _includes/payment-order-checkout-standard.md:6]: Sure. If too many `if` statements are required, keeping the examples separate may be easier.

**Review by asbjornu — APPROVED** [review]
👏🏼

**Pull request merged**